JokeRandomizer/JokeRandomizer/CustomButtons/CustomButton.swift

```diff
@@ -21,11 +21,12 @@ class CustomButton: UIButton {
     
     func setupButton() {
         setShadow()
-        
+        shake()
+        let image = UIImage(systemName: "arrow.clockwise.circle.fill")
+        setImage(image, for: .normal)
         setTitleColor(.white, for: .normal)
-        
         backgroundColor = .blue
-        titleLabel?.font = UIFont(name: "AvenirNext-DemiBold", size: 10)
+//        titleLabel?.font = UIFont(name: "AvenirNext-DemiBold", size: 10)
```

Comment:
remove unused codes

Reply:
Removed unused code and Custom Button have a init() for favorite, refresh and list button